packages/composites/vesper/src/index.ts

```diff
@@ -1,12 +1,12 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { endpointSelector, makeExecute } from './adapter'
-import { makeConfig } from './config'
+import { makeConfig, NAME } from './config'
 
-const NAME = 'VESPER'
+const adapterContext = { name: NAME }
 
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute())
export { NAME, makeConfig, makeExecute, server }
```